presto-product-tests/conf/tempto/tempto-configuration.yaml

```diff
@@ -3,3 +3,23 @@ databases:
     host: hadoop-master
   presto:
     host: presto-master
+
+  prestoConfigUserAlice:
+    host: localhost
+    server_address: ${databases.presto.host}:8080
+    cli_server_address: ${databases.presto.host}:8080
```

Comment:
Removed. I had added this field temporarily for other tests and forgot to remove it.